cocaine/tools/actions/auth.py

```diff
@@ -8,124 +8,141 @@
 from cocaine.decorators import coroutine
 
 from . import Action
-from .. import actions
 
 
 COLLECTION_TOKENS = 'tokens'
 COLLECTION_GROUPS = 'auth-groups'
 
 
-class List(actions.List):
+class Auth(object):
+    """Auth groups is a lie"""
     def __init__(self, storage):
-        super(List, self).__init__(COLLECTION_GROUPS, ['auth'], storage)
-
-
-class Create(Action):
-    def __init__(self, storage, name, token, force=False):
-        self._name = name
-        self._token = token
-        self._force = force
-        self._storage = storage
+        self._storage = storage  # HINT: storage.write(namespace, key, value, tags)
 
     @coroutine
-    def execute(self):
-        if not self._force:
+    def create_group(self, group_name, token, force=False):
```

Comment:
You can't just rename arguments without synchronizing with https://github.com/cocaine/cocaine-tools/blob/master/cocaine/tools/dispatch.py.